decidim-proposals/app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -40,12 +43,20 @@ def order_by_votes?
 
         def reorder(proposals)
           case order
-          when "random"
-            proposals.order_randomly(random_seed)
+          when "most_commented"
+            proposals.left_joins(:comments).group(:id).order(Arel.sql("COUNT(decidim_comments_comments.id) DESC"))
```

Comment:
What's the difference with the following suggested snippet?
```suggestion
            proposals.left_joins(:comments).group(:id).order("COUNT(decidim_comments_comments.id) DESC")
```

Why use `Arel.sql`?